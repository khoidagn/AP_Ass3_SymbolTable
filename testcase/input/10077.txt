INSERT x string
ASSIGN x _x