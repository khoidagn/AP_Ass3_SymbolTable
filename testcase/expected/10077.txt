Invalid: ASSIGN x _x